src/BaseRepository.js

```diff
@@ -52,6 +53,14 @@ class BaseRepository {
     return this.mapper.createEntity(assignedValues, true)
   }
 
+  updateBy (entityInstance, fields, relationshipFields = {}, whereFields = []) {
```

Comment:
I just don't like the method's name, maybe `updateFields` would be better?

Reply:
Yeah, I did this change.